src/model/AuthUser.js

```diff
@@ -83,4 +83,11 @@ export default class AuthUser extends Ressource {
       email_address: emailAddress
     });
   }
+  static async getUserIdFromUsername(username) {
```

Comment:
`getUserByUsername()`?

Reply:
We have the same function in AccountProfile so maybe he tries to be consistant here, but we need to rename both

Reply:
Resolved with 3f9ec93.